src/applications/post-911-gib-status/containers/StatusPage.jsx

```diff
@@ -78,7 +78,7 @@ class StatusPage extends React.Component {
             ), Monday &#8211; Friday, 8:00 a.m. &#8211; 7:00 p.m. ET
           </div>
         </div>
-      </div>
+      </>
```

Comment:
I tested this whole user flow with a logged-in user to be sure I didn't break anything. This page was super skinny, so I removed the grid classes.